features/collection/components/FilterList/FilterRow.tsx

```diff
@@ -67,16 +153,21 @@ export const FilterRow = React.memo(
                         color="gray.500"
                         style={{
                            transform: `rotate(${isExpanded ? '45' : '0'}deg)`,
-                           transition: 'transform 300ms',
+                           transition: `transform ${TOGGLE_ANIMATION_DURATION_MS}ms`,
```

Comment:
Why did we change this from 300 -> 250, and on line 143 we left the background color transition at 300ms?
`transition="background-color 300ms"`

Reply:
@masonmcelvain because they're two different animations.. normally I would stick to 300ms, but for the slide-up slide-down I opted for a faster time since it helps hiding some tricks that makes the animation work